iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/SectionModification.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.modification;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.commons.reporter.Reporter;
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.ShuntCompensator;
+
+import java.util.Objects;
+
+/**
+ * Simple {@link NetworkModification} to change the section of a shunt compensator.
+ *
+ * @author Nicolas PIERRE <nicolas.pierre at artelys.com>
+ */
+public class SectionModification extends AbstractNetworkModification {
```

Comment:
In the actions used in the security analysis API, we have chosen `ShuntCompensatorPositionAction`. I think finally we have to chose a name close to this first choice to remain coherent and uniform.